roles/config-ipa-client/defaults/main.yml

```diff
@@ -3,3 +3,5 @@
 move_local_user_home: False
 new_local_home_dir: "/lclhome"
 temporary_username: "lcluser"
+
+ipa_client_install: true
```

Comment:
@kenhitchcock Can we change this to false? I'd like to align with what the default is in the task (and also prefer to be explicit about wanting to install packages)

Reply:
Changed default to false, its up to the user to set the variable if they wish to deploy.